Makefile

```diff
@@ -27,7 +35,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # IMAGE defines the base image to be used for operator, bundle and catalog.
-IMAGE ?= quay.io/redhat-developer/gitops-operator
+IMAGE ?= quay.io/nittalaakhil/openshift-gitops-operator
```

Comment:
This needs to be reverted. For your testing, you can try setting the variable in environment. Like `IMAGE=quay.io/nittalaakhil/openshift-gitops-operator make build`